ops/nomad.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
If this was sourced from somewhere else it could be good to reference it here so people understand any extra context.

Reply:
Sorry, "sourced" was a bad term here, I meant if you had copied the script from elsewhere then it would be good to add the URL here. But also just a docstring on the purpose would be good too.

Reply:
Done 👍

Reply:
And some more on the what the thing is actually for: https://github.com/cljdoc/cljdoc/pull/248/commits/01e6cbe9e8198723bca5e6446cc6febe2c0121eb